JustSaying.Tools/JustSaying.Tools.csproj

```diff
@@ -9,6 +9,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Magnum" Version="2.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
Should these versions also be lower?

Reply:
Do we ship this? I thought it was an executable tool not a library, so would want the latest versions.

Reply:
Ah, that's correct, it's not referenced by `JustSaying`